yb_os_maint/yb_os_maint.py

```diff
@@ -1497,36 +1517,44 @@ def verify(self):
         errs = 0
 
         if self.node_json['state'] == 'Live':
+            passed, message = self.Compare_node_service_status_to_YBA('Master')
             if self.node_json['isMaster']:
                 log('   YBA shows node as having a Master - checking for process')
-                passed, message = self._compare_node_service_status_to_YBA('Master')
                 if passed:
                     log('     Check passed: master process found on node')
                 else:
                     log(message, True)
                     errs += 1
             else:
-                log('   YBA shows node as NOT having a Master - skipping check')
+                if passed:
+                    log('   ERROR: master process found on node, but YBA does not show node as being a Master', True)
+                    errs += 1
+                else:
+                    log('   YBA shows node as NOT having a Master - and no master is running')
```

Comment:
Is the logic here inverted? Wouldn't we want the check to pass?

If the logic is correct, the variable `passed` should probably have a name that more clearly expresses the intent of the test (`is_running`?).